app/code/Bulbulatory/Recommendations/Controller/Customer/SendRecommendation.php

```diff
@@ -0,0 +1,176 @@
+<?php
+
+namespace Bulbulatory\Recommendations\Controller\Customer;
+
+use Bulbulatory\Recommendations\Exception\RecommendationException;
+use Bulbulatory\Recommendations\Helper\Acl;
+use Exception;
+use http\Exception\InvalidArgumentException;
+use Throwable;
+use Magento\Framework\App\Action\Action;
+use Magento\Framework\App\Action\Context;
+use Bulbulatory\Recommendations\Model\RecommendationRepository;
+use Psr\Log\LoggerInterface;
+use Magento\Customer\Model\Session;
+use Bulbulatory\Recommendations\Helper\Email;
+
+class SendRecommendation extends Action
+{
+    const EMAIL_TEMPLATE  = 'recommendations/general/recommendation_email';

```

Comment:
The email template config path is not really related to this class - it shouldn't care what the template is, it should just call a method that sends the email. Better place for this would be the config class, then in the method fetching template you can use it.

Reply:
@TM18

> The email template config path is not really related to this class - it shouldn't care what the template is, it should just call a method that sends the email. Better place for this would be the config class, then in the method fetching template you can use it.

Fixed!